bigquery/indexer.py

```diff
@@ -462,15 +467,26 @@ def main():
     bq_client = bigquery.Client(project=deploy_project_id)
     storage_client = storage.Client(project=deploy_project_id)
 
+    participant_docs = {}
+    field_docs = {}
     for table_name in bigquery_config['table_names']:
         table = read_table(bq_client, table_name)
-        index_fields(es, fields_index_name, table, sample_id_column)
+        field_docs = add_table_to_fields_docs(es, fields_index_name, table,
+                                              sample_id_column, field_docs)
         create_mappings(es, index_name, table_name, table.schema,
```

Comment:
Let's move create_mappings before add_table_to_fields_docs(), so  add_table_to_fields_docs() is next to add_table_to_participant_docs()